main.js

```diff
@@ -0,0 +1,43 @@
+function createCalendar(id, year, month) {
+  const container = document.getElementById(id);
+  const currentMonth = month - 1;
+  const date = new Date(year, currentMonth);
+  const days = ['Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб', 'Нд'];
+  let table = '<table><tr>';;
+
+  for (let i = 0; i < days.length; i++) {
+    table += `<th>${days[i]}</th>`
+  }
+  table += '</tr><tr>';
+
+  for (let i = 0; i < getDay(date); i++) {
+    table += '<td></td>';
+  }
+
+  while (date.getMonth() === currentMonth) {
```

Comment:
For my mind will be better to calc month length before, and use for. Also in implementation which I propose, you do not need to use the date as a state(means the state of application/function/loop).